### ISSUE TITLE
KL10/1140 panel

### ISSUE THREAD

**Issue by larsbrinkhoff** [thorhalbert/DEC-VR#1]
Having recently drawn my own KL10 11/40 panel, I noticed some differences between your drawings and the reference photo.  Your drawing has some similarities to the original 11/40.

- The "KL10" font.  The .png "K" doesn't join symmetrically in the middle.  The .svg has a serif font.
- The original 11/40 "A" has the horizontal bar higher up, but the KL10 is quite low.
- The original 11/40 "R" has a straight line, but the KL10 has a bent line.  Detail here: https://raw.githubusercontent.com/larsbrinkhoff/pidp-kl10/master/pics/PDP-10-Model-KL-10-Digital-Equipment-Corporation-006_book_f.jpg
- The original 11/40 has a narrow space between LOAD and ADR, KL10 has a wide space.
- The original 11/40 has a gap between ENABLE and HALT, KL10 has no gap.

![kl10-diffs](https://github.com/thorhalbert/DEC-VR/assets/775050/ef7cf1dd-5ff8-46d8-86f8-550cf4f7061f)

**Comment by thorhalbert** [comment]
I noticed that font thing.   I suspect it was using the wrong font since I did it a long time ago.  I will likely have to draw that 'K' manually.   Generally DEC stuff is helvetica, but not really for some things--here though it's doing some font such as times.  I'll have to load it up in inkscape and see if it can generate it correctly -- once upon a time it rendered correctly.

**Comment by larsbrinkhoff** [comment]
Yes, Helvetica looks right with regards to upper-case A and R.  I drew the letter shapes myself with Bezier curves, phew.

**Comment by thorhalbert** [comment]
Yeah, that's what I had to do with the PDP-11 logos for the 70 (hand draw them).   Ariel is almost, but not quite Helvetica.
